packages/cli/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apt-get update
 RUN apt-get install -y nodejs
 
 # Install sharp TODO update this when we change sharp versions
-RUN npm install sharp@0.33.0
+RUN npm install sharp@0.33.0 lerc@^4.0.4
```

Comment:
where in the CLI is this needed?

Reply:
We use this cli container for the server in [basemaps-config](https://github.com/linz/basemaps-config/blob/master/scripts/bms.sh). So we need to install lerc this into cli container.

Reply:
if the server is being installed it should come with `lerc` so I think the server is just missing the lerc dep? how is `sharp` defined in basemaps/server ?

Reply:
`sharp` is installed as dependency already in the [basemaps/server](https://github.com/linz/basemaps/blob/master/packages/server/package.json#L50)

But we are not installing `sharp` in the basemaps/cli, so we installed into container directly.

Reply:
I don't think server package is installed in the container, it is just a [function call](https://github.com/linz/basemaps/blob/master/packages/cli/src/cli/server/action.serve.ts) to the `package/server`, so we still need to install `lerc` in the cli container.